utils/python/tests/scripts_regression_tests

```diff
@@ -12,8 +12,8 @@ from  CIME.XML.machines import Machines
 from  CIME.XML.files import Files
 from  CIME.case import Case
 
-SCRIPT_DIR = CIME.utils.get_acme_scripts_root()
-
+SCRIPT_DIR = CIME.utils.get_scripts_root()
+TOOLS_DIR = os.path.join(SCRIPT_DIR,"Tools")
```

Comment:
What's the difference between a tool and a script? I think separating them is fine, I'd just like to know what the difference is.